2363/1/triangle.rb

```diff
@@ -0,0 +1,28 @@
+# Triangle Project Code.
+
+# Triangle analyzes the lengths of the sides of a triangle
+# (represented by a, b and c) and returns the type of triangle.
+#
+# It returns:
+#   :equilateral  if all sides are equal
+#   :isosceles    if exactly 2 sides are equal
+#   :scalene      if no sides are equal
+#
+# The tests for this method can be found in
+#   about_triangle_project.rb
+# and
+#   about_triangle_project_2.rb
+# This method smells of :reek:UtilityFunction
+# This method smells of :reek:FeatureEnvy
+# This method smells of :reek:TooManyStatements
+def triangle(*sides)
```

Comment:
Hi. I'm a mentor from Rubizza. Alena asked to help a bit with reviews, so here am I :)

Interesting approach to solve the task, unusual indeed 👍